Shared/Samples/Set visibility of subtype sublayer/SetVisibilityOfSubtypeSublayerView.Model.swift

```diff
@@ -0,0 +1,166 @@
+        /// The status text to display to the user.
+        @Published private(set) var statusText = ""
+        
+        /// The  subtype sublayer's label definition.
+        private(set) var labelDefinition: LabelDefinition = {
+            // Make and stylize the text symbol.
+            let textSymbol = TextSymbol()
+            textSymbol.backgroundColor = .clear
+            textSymbol.outlineColor = .white
+            textSymbol.color = .blue
+            textSymbol.haloColor = .white
+            textSymbol.haloWidth = 2
+            textSymbol.size = 10.5
+            // Make a label definition and adjust its properties.
+            let labelExpression = SimpleLabelExpression(simpleExpression: "[nominalvoltage]")
+            let labelDefinition = LabelDefinition(labelExpression: labelExpression, textSymbol: textSymbol)
+            labelDefinition.placement = .pointAboveRight
+            labelDefinition.usesCodedValues = true
+            return labelDefinition
+        }()
+        
+        // The formatter used to generate strings from scale values.
+        private let scaleFormatter: NumberFormatter = {
+            let numberFormatter = NumberFormatter()
+            numberFormatter.numberStyle = .decimal
+            numberFormatter.maximumFractionDigits = 0
+            return numberFormatter
+        }()
+        
+        /// A Boolean value indicating whether the settings should be presented.
+        @Published var isShowingSettings = false
+        
+        /// A Boolean value indicating whether to show the subtype sublayer.
+        @Published var showsSublayer = true
+        
+        /// A Boolean value indicating whether to show the subtype sublayer's renderer.
+        @Published var showsOriginalRenderer = true
+        
+        /// The current scale of the map.
+        @Published var currentScale: Double = .zero
+        
+        /// The map's current scale value in text.
+        @Published var currentScaleText: String = ""
+        
+        /// The subtype sublayer's minimum scale value in text.
+        @Published var minimumScaleText: String = "Not Set"
+        
+        init() {
+            map.initialViewpoint = .initialViewpoint
+            subtypeFeatureLayer = SubtypeFeatureLayer(featureTable: featureTable)
+            subtypeFeatureLayer.scalesSymbols = false
+        }
+        
+        deinit {
+            ArcGISEnvironment.authenticationManager.arcGISCredentialStore.removeAll()
+        }
+        
+        /// Performs important tasks including adding credentials, loading and adding operational layers.
+        @MainActor
+        func setup() async {
+            do {
+                try await ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(.publicSample)
+                try await subtypeFeatureLayer.load()
+                map.addOperationalLayer(subtypeFeatureLayer)
+                subtypeSublayer = subtypeFeatureLayer.sublayer(withSubtypeName: "Street Light")
```

Comment:
This can be guarded, as it is optional chained 3 times below. Implicitly unwrapped optional or not is up to you.

Reply:
With `guard` the return will execute and then the subtype sublayer's `isVisible` and `renderer` properties can't be updated. It seems unsafe to force unwrap.

Reply:
Hmm I don't quite get it. If `subtypeSublayer` is `nil`, then all the optional chainings will stop at the `nil`. It seems to me that all the settings UI should be disabled if it is `nil`.

Reply:
Okay I unwrapped the layer and also made it an implicitly unwrapped optional which will call `fatalError()` if not initialized. I can make the layer optional again if preferred.

Reply:
Thank you. I'm 50/50 on whether or not to use IUO. The reasons that it may become `nil` as I can think of are

- `subtypeFeatureLayer` is not loaded due to auth or network issues. `fatalError` might not be the best here. However, I have good faith in sampleserver7's stability.
- `sublayer(withSubtypeName:)` method returns `nil` due to a programmer error of using a wrong name. `fatalError` in this case makes sense.

Reply:
What is the variable in question about the IUO?  If it is [this variable](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/205/files#diff-ab27dee8370a163cb44caef7203e764ac165300ec47dcb7ad31b1fbcc00b427eR32), then I don't think it should be IUO.

This line could throw:
```
try await subtypeFeatureLayer.load()
```

and then the variable `subtypeSublayer` would be `nil` and unsafe.  It needs to be an optional.

Reply:
Fixed.